tests/llama_index_sample.py

```diff
@@ -84,12 +84,39 @@
 #             "trace_id": "0xbd54e5d0edcd96634fa8a02c25c27519",
 #             "parent_id": "None",
 #             "attributes": {
-#                 "workflow_input": "What did the author do growing up?",
-#                 "workflow_name": "llama_index_1",
-#                 "workflow_output": "The context does not provide information about what the author did while growing up.",
-#                 "workflow_type": "workflow.llamaindex"
-#             },
-#             "events": []
+#                    "tags": [
+#                       "text-embedding-3-large",
+#                        "ChromaVectorStore"
+#                    ],
+#                   "type": "vector_store",
+#                   "provider_name": "ChromaVectorStore",
+#                   "embedding_model": "text-embedding-3-large",
```

Comment:
could you confirm if vector store related attributes are added in root span or retrieval related span

in this llama index eg its in root span(parent=None) and in langchain eg above its in span langchain.task.VectorStoreRetriever, I think

Reply:
done , added to retrieval span.